drizzlepac/haputils/poller_utils.py

```diff
@@ -255,7 +255,7 @@ def interpret_mvm_input(results, log_level, exp_limit=2.0):
 
     log.debug("Interpret the poller file for the observation set.")
     obset_table = build_poller_table(results, log_level, 
-                                     poller_type='mvm')
+                                     poller_type='svm')
 
```

Comment:
This seems to be in the MVM portion of the code.  It seems the poller_type should stay as 'mvm'.

Reply:
Resolved in commit [fe90129](https://github.com/spacetelescope/drizzlepac/pull/682/commits/fe901293d2d6993a96db29d1b44bc0d6f1497c35)